lib/index.js

```diff
@@ -45,6 +45,12 @@ async function run() {
 
   const npmVersion = await core.group(`Update npm to ${NPM_VERSION}`, () => updateNpm(NPM_VERSION));
 
+  const path = core.getInput("path");
+  if (path !== "./") {
+    core.info(`Change directory to ${path}`);
+  }
+  process.chdir(path);
```

Comment:
[suggestion] I want to avoid repeating the default path. How about using `process.cwd()` instead? For example:

```suggestion
  process.chdir(core.getInput("path"));
  core.info(`Current directory: ${process.cwd()}`);
```